hello-slack/app.py

```
@@ -0,0 +1,378 @@
+                 if len(formatted_transcription) > 3800 :
+                      say(f"<@{user}> 話者分離結果(冒頭部分):\n{formatted_transcription[:3800]}...", thread_ts=thread_ts)
+                      logging.warning("Diarization result was too long, truncated.")
+                 else:
+                      say(f"<@{user}> 話者分離結果:\n{formatted_transcription}", thread_ts=thread_ts)
+            else:
+                 logging.error("Speaker diarization failed or returned empty.")
+                 say(f"<@{user}> 話者分離に失敗しました。", thread_ts=thread_ts)
+
+
+        except requests.exceptions.RequestException as e:
+            logging.error(f"Error downloading file: {e}")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)